aws-ami-creation/hcl2-files/variables.pkr.hcl

```diff
@@ -0,0 +1,23 @@
+variable "region" {
+  default     = "us-east-1"
+  description = "The AWS region you're using"
+  type        = string
+}
+
+variable "instance_type" {
+  default     = "trn1.2xlarge"
+  description = "EC2 machine type for building AMI"
+  type        = string
+}
+
+variable "source_ami" {
+  default     = "ami-0328507805a665ec1"
```

Comment:
can we derive this automatically to use the latest released AWS image? then we don't need to check, e.g. with the AWS CLI you can do 
```bash
aws ec2 describe-images --region us-east-1 --owners amazon --filters 'Name=name,Values=Deep Learning AMI Neuron PyTorch 1.13 (Ubuntu 20.04) ????????' 'Name=state,Values=available' --query 'reverse(sort_by(Images, &CreationDate))[:1].ImageId' --output text
```

Reply:
I think we should keep the ability to regenerate previous AMIs, so it is IMHO better to have the exact version of the source AMI we used somewhere.
In other words, if we can put that in another file that calls this, then I am fine with using another default. Otherwise I would keep it.

Reply:
I would also agree to keep the exact version, as it will help in debugging if we ran into any kind of problem in future.

Reply:
SGTM! But then lets include it in the "github" actions as parameter that you could overwrite it if needed.